spec/models/user_spec.rb

```diff
@@ -29,4 +29,16 @@
       end
     end
   end
+  describe "Instance Methods" do
```

Comment:
```suggestion

  describe "Instance Methods" do
```

Reply:
Rubocop didn't like this, so closing it.